devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/directive-forest/directive-forest.component.html

```diff
@@ -39,6 +39,12 @@
       @if (isSelected(node)) {
         <span class="console-reference"> == $ng0 </span>
       }
+
+      @if(node.hydration === 'hydrated') {
+        <span class="hydration">hydrated</span>
+      } @else if(node.hydration === 'skipped') {
+        <span class="hydration">hydration skipped</span>
```

Comment:
If it will be used only to showcase hydration status, can we just have **skipped**? And maybe change the color to orange or yellow? And maybe add a tooltip to show a clear message.